app/controllers/review_item/accept.js

```diff
@@ -60,38 +113,46 @@ export default Ember.Controller.extend({
 
     // load default packages
     if (itemType && packages.length === 0) {
-      itemType.get("defaultChildPackagesList").apply(itemType)
-        .forEach(t => this.send("addPackage", t.get("id")));
+      this.loadDefaultPackages(itemType);
     }
   }),
 
-  cannotSave(){
-    var pkgs = this.get('itemPackages');
-    if(pkgs && pkgs.length > 0 && (pkgs.get("firstObject.hasAllPackagesDesignated") || pkgs.get("firstObject.hasAllPackagesDispatched"))){
-      this.get('messageBox').alert(this.get("i18n").t('designated_dispatched_error'));
+  cannotSave() {
+    var pkgs = this.get("itemPackages");
+    if (
```

Comment:
move this logic in some separate function or add it in some variable and used it in `if` condition. Intention is not very clear here.